app/src/main/java/org/simple/clinic/scheduleappointment/ScheduleAppointmentSheetController.kt

```diff
@@ -191,16 +195,17 @@ class ScheduleAppointmentSheetController @AssistedInject constructor(
         .replay()
         .refCount()
 
-    val appointmentStream = schedulingSkippedStream.map {
-      val currentFacilityUuid = currentFacility.get().uuid
+    val appointmentStream = schedulingSkippedStream
+        .map {
+          val currentFacilityUuid = currentFacility.get().uuid
```

Comment:
We do not seem to be using a `Scheduler` here, shouldn't we need it here?

Reply:
Does not seem to be necessary, I tapped on the sheet over 20 times trying to trigger it. In any case, this will be fixed once the sheet is fully ported to Mobius, so 🤷